packages/perseus/src/widgets/radio/base-radio.jsx

```diff
@@ -340,9 +347,18 @@ class BaseRadio extends React.Component<$FlowFixMe, $FlowFixMe> {
     };
 
     focus: (number) => boolean = (i) => {
-        // $FlowFixMe[incompatible-use]
-        // $FlowFixMe[prop-missing]
-        ReactDOM.findDOMNode(this.refs["radio" + (i || 0)]).focus(); // eslint-disable-line react/no-string-refs
+        const ref = this.choiceRefs[i || 0];
+        // note(matthew): we know this is only getting passed
+        // to a WB Clickable button, so we force it to be of
+        // type HTMLButtonElement
+        const anyNode = (ReactDOM.findDOMNode(ref.current): any);
+        const castNode = (anyNode: ?HTMLButtonElement);
+
+        if (castNode) {
+            castNode.focus();
+        } else {
+            return false;
```

Comment:
Possible API change for Choice: before I think it would throw if the node didn't exist, now it returns false. I don't think this is an issue though.

Reply:
Question (non-blocking): So there isn't a time when an error would be useful from this? Does the false returned here ever get used for anything? If neither the error nor the false return are used, is an else section necessary?

Reply:
@Myranae The more I think about it, the less I feel there's ever going to be a chance for this to fail/error. It should always end up being a Clickable which should have some kind of focusable element.